index.js

```diff
@@ -3,32 +3,34 @@ var bodyParser = require('body-parser')
 var pkg = require('./package')
 var _ = require('lodash')
 var multipart = require('connect-multiparty')()
-var koop = require('./lib')
+var lib = require('./lib')
 
 module.exports = function (config) {
-  var app = express()
+  var koop = express()
 
-  /**
-   * koop setup
-   */
+  // inherit everything from lib ¯\_(ツ)_/¯
+  for (var method in lib) {
+    if (lib.hasOwnProperty(method)) koop[method] = lib[method]
+  }
 
-  koop.config = config = config || {}
-  koop.log = new koop.Logger(config)
+  koop.config = config || {}
+  koop.log = new koop.Logger(koop.config)
```

Comment:
Would be better if Logger wasn't hanging off of koop. I was going to propose removing constructors from being exposed on the koop object in a future major version bump.